www/perl-lib/SMT/Rest/SCCAPIv4.pm

```diff
@@ -348,6 +351,86 @@ sub delete_system
     return (Apache2::Const::OK, {});
 }
 
+sub delete_single_product
+{
+    my $self = shift || return (undef, undef);
+
+    # We are sure, that user is a system GUID
+    my $guid = $self->user();
+    my $c    = JSON::decode_json($self->read_post());
+
+    foreach my $param ( qw{identifier version arch} ) {
+        unless ( exists $c->{$param} && $c->{$param} ) {
+            return (Apache2::Const::HTTP_UNPROCESSABLE_ENTITY, "Missing required parameter: $param");
```

Comment:
Perhaps, but this API is used only by SUSEConnect, so if those parameters aren't present -- something must have gone wrong. I guess I'm going to leave it as is for now.